### ISSUE TITLE
jpeg images with trailer data are reported as incomplete

### ISSUE THREAD

**Issue by Mycah** [waikato-datamining/python-image-complete#2]
```
>>> from image_complete.jpg import is_jpg_complete, is_jpg
>>> is_jpg_complete('10-o.jpg')
False
```
but if i run:
```
exiftool -trailer:all= 10-o.jpg
    1 image files updated
```
then:
```
>>> is_jpg_complete('10-o.jpg')
True
```

**Comment by fracpete** [comment]
At the moment, I don't consider this a bug. There shouldn't be junk data after the EOF marker. :-)
I'd have to scan the whole file for the EOF marker opposed to just reading the last two bytes.
I might introduce a *lenient* flag though that would allow junk data, but also be slower.

**Comment by Mycah** [comment]
I discovered specifically what my issue is.  The site I'm downloading images from seems to have a 0xa0 (`\n`) byte at the end of several (not all) of their jpeg files for some reason.  I modified the jpeg detection to scan for the last 3 bytes, and check for 0xFF and 0xD8 there instead.

I hope it isn't a common scenario.

**Comment by fracpete** [comment]
I've just released [version 0.0.5](https://github.com/waikato-datamining/python-image-complete/releases/tag/v0.0.5) which allows you to run in **lenient mode**, i.e., tolerating trailing junk data. For example:
```python
from image_complete.auto import is_image_complete
is_image_complete("/some/where/hello_world.jpg", strict=False, check_size=100)
```

Notes:
* `strict=False` - turns off strict mode
* `check_size=100` - looks through last 100 bytes for EOF marker byte(s); only applies to gif, jpg and png files
